### DISCUSSION TITLE
Reset password

### DISCUSSION THREAD

**Discussion by koksikus** [directus/directus#9471] [Feature Requests]
In the case of resetting the password, when the user exists in the database, there is information that an e-mail with a link to change the password has been sent.  The same situation occurs when the user is not in the database and the e-mail is not sent.  It should say check your e-mail address.

**Comment by benhaynes** [comment]
This is for security reasons, so no one can ping the reset endpoint to see who is a user of the system.

**Reply by koksikus** [reply]
I didn't think about it, actually a good approach.  Only maybe the message should be different, e.g. if you have entered your email correctly, we will send you a reset email ... Which will be 100% true.  Or can replace the email login with a standard login and the email address should be provided only during registration... Only idea

**Reply by rijkvanzanten** [reply]
Good point on the messaging! Wanna open a PR for that tweak? Here's the source for that:

https://github.com/directus/directus/blob/2cf9315b94cf0bf587b94be36aa2ec857002383e/app/src/lang/translations/en-US.yaml#L530

**Reply by koksikus** [reply]
I don't know what you mean with PR or how to do such things... maybe some hint 😉 even for the future...

Something like this should be fine what you think

"If you entered your email correctly, we've sent you a secure link to reset your password"

**Reply by koksikus** [reply]
@rijkvanzanten sorry if i bother you but could you please give me a hint. I'd like to do this (above).

**Reply by rijkvanzanten** [reply]
The easiest way to create a PR is by editing the file on GitHub directly!:

![CleanShot 2021-11-24 at 20 57 47@2x](https://user-images.githubusercontent.com/9141017/143364475-df702579-8c5e-4e5b-8720-b6bb8a3ef7cd.png)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
